src/lib/load-breach.js

```diff
@@ -164,9 +166,25 @@ async function loadBreachLayer (breachId, layerName) {
     })
   })
 
-  const d = await resp.json()
-  const data = JSON.parse(d.d)
-  // get  the first layerset  if available,  otherwise return null
+  const filterFetch = await fetch(FILTERS_API_URL, {
+    method: 'GET',
+    mode: 'cors',
+    credentials: 'omit',
+    headers
+  })
+
+  const [breachData, filtersData] = await Promise.all([breachFetch, filterFetch])
+    .then(responses => responses.map(response => response.json()))
+
+  const properties = await filtersData
```

Comment:
is this `await` needed?

Reply:
Yes, otherwise it will return as `Promise {<pending>}`.